esx_service/vmci/connection_types.h

```diff
@@ -29,6 +29,13 @@
 #define START_NON_PRIVILEGED_PORT 1024
 
 /*
+ * Check and set errno helper
+ * Useful when send/recv gets us less than we wanted, abd we want to set errno
```

Comment:
This actually bring it back to the original retry login and intent. 
There is a short-term guard in place (put in a few weeks ago) to catch the case
```golang
else {
			msg = fmt.Sprintf("Internal issue: ret != 0 but errno is not set. Cancelling operation - %s ", C.GoString(&ans.errBuf[0]))
		}
```
but the current one gets us back to the the original retry logic - which is to ignore the current exchange, reopen the socket and reissue command. This PR does not change any of that,